src/test/java/conan/commands/ListProfilesTest.java

```diff
@@ -20,8 +20,8 @@
 
     @Test
     public void testListProfiles() {
-        verifyProfiles(Sets.newHashSet());
+        //verifyProfiles(Sets.newHashSet());
```

Comment:
Why were these tests removed?

Reply:
Thanks, forgot to uncomment after the refactor